greedy_algorithms/kruskals_minimum_spanning_tree.cpp

```diff
@@ -55,15 +55,15 @@ void findMinimumEdge(const int &infinity,
  * @returns 0 on exit
  */
 int main() {
-    constexpr int INFINITY = 99999;
-    std::array<std::array<int, 6>, 6> graph{
-        0,        4,        1,        4,        INFINITY, INFINITY,
-        4,        0,        3,        8,        3,        INFINITY,
-        1,        3,        0,        INFINITY, 1,        INFINITY,
-        4,        8,        INFINITY, 0,        5,        7,
-        INFINITY, 3,        1,        5,        0,        INFINITY,
-        INFINITY, INFINITY, INFINITY, 7,        INFINITY, 0};
+    constexpr int POSITIVE_INFINITY = 99999;
```

Comment:
why not use `numeric_limits<int>::infinty();` from the cmath library?

Reply:
numeric_limits<int>::infinty() returns 0

Reply:
Ah I forgot infinity only works for floats. What about numeric_limit<int>::max()?